### PR TITLE
x11-misc/kdocker: version bump to 5.1

### PR DESCRIPTION
**Pull request by realnc**
Build log: https://648574.bugs.gentoo.org/attachment.cgi?id=523840

Closes: https://bugs.gentoo.org/648574
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: x11-misc/kdocker/kdocker-5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	dev-qt/qtx11extras:5
+	x11-libs/libX11
+	x11-libs/libXmu
+	x11-libs/libXpm
+"
+RDEPEND="${DEPEND}"
+
+DOCS=( AUTHORS BUGS ChangeLog CREDITS README.md TODO )
+
+S=${WORKDIR}/KDocker-${PV}
```
Comment (mgorny) [inline x11-misc/kdocker/kdocker-5.1.ebuild]: Minor, optional suggestion: declare `MY_P=KDocker-${PV}` and name the distfile `${MY_P}.tar.gz`. It's generally preferable if the archive and the directory inside it are named exactly the same.

Comment (realnc) [reply x11-misc/kdocker/kdocker-5.1.ebuild]: Done.

**Comment by realnc** [comment]
@mgorny 

> Would you be interested in becoming a maintainer for this package? If so, please add yourself and proxy-maint to metadata.xml file.

I can maintain the package (it's an important package for me.)

PR updated.

**Comment by mgorny** [comment]
Two more problems (I would suggest upgrading Portage if you haven't seen them):

```
 * Messages for package x11-misc/kdocker-5.1:
 * Log file: /var/log/portage/x11-misc:kdocker-5.1:20180315-175253.log

 * Problems with installed bash completions were found:
 * 
 * 	kdocker: 'have' command is deprecated and must not be used.
 * 
 * For more details on installing bash-completions, please see:
 * https://wiki.gentoo.org/wiki/Bash/Installing_completion_files
 * 
 * QA Notice: new icons were found installed but GTK+ icon cache
 * has not been updated:
 *   /usr/share/icons/hicolor/128x128/apps/kdocker.png
 * Please make sure to call gnome2_icon_cache_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.

**Comment by realnc** [comment]
@mgorny 
> Problems with installed bash completions were found:
> kdocker: 'have' command is deprecated and must not be used.

This is an upstream issue. I can't do anything about it for this version bump as I am completely ignorant about bash completion, it's syntax or how it works. But I'll look into it for the future.

> QA Notice: new icons were found installed but GTK+ icon cache
> has not been updated:

That's fixed now.

**Pull request closed without merging**